### PR TITLE
feat: added darkmode

### PR DESCRIPTION
**Pull request by Roshannair24**
added darkmode
CPT2202212347-1503x656 (1)

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: css/styles.css [hunk 1]
```diff
@@ -832,6 +832,32 @@ body {
   height: 100vh;
 }
 
+.wallpaper-base.dark-background {
+  background: rgba(255, 255, 255, 0);
```
Comment (amarphule) [inline css/styles.css:836]: use color variable

File: css/styles.css [hunk 2]
```diff
@@ -832,6 +832,32 @@ body {
...
+  background: linear-gradient(
+    90deg,
+    rgba(99, 7, 7, 0) 0%,
+    rgba(240, 226, 217, 0) 100%
```
Comment (amarphule) [inline css/styles.css:840]: same

File: index.js [hunk 3]
```diff
@@ -15,3 +14,47 @@ function reportWindowSize() {
...
+    ?.classList.toggle("dark-background");
+  document
+    .querySelector(".installation-guide-container")
+    ?.classList.toggle("dark-background");
+
+  document
+    .querySelector(".installation-guide-text")
+    ?.classList.toggle("dark-background");
+
+  // document.querySelector("#list-iframe-v-container-leftside").classList.toggle("light-background");
```
Comment (amarphule) [inline index.js:45]: remove comment

File: index.js [hunk 4]
```diff
@@ -15,3 +14,47 @@ function reportWindowSize() {
...
+
+  sidebarBtnArr.forEach((currentValue, index, arr) => {
+    currentValue.classList.toggle("light-foreground");
+  });
+
+  notifArr.forEach((currentValue, index, arr) => {
+    currentValue.classList.toggle("light-background");
+  });
+
+  // document.querySelectorAll(".k-link").classList.toggle("light-foreground");
```
Comment (amarphule) [inline index.js:59]: remove comment

**Review by amarphule — APPROVED** [review]
nice @Roshannair24 , code is well formatted. given some suggestion please take a look

**Pull request merged**